bin/docker-build.js

```diff
@@ -860,9 +1043,16 @@ function importImageSnapshotsIntoImgapi(builder, opts, callback) {
            }
         ], function importCleanup(err) {
             if (err) {
-                log.error('Error importing images into IMGAPI', err);
+                log.error('Unable to import image into imgapi: %s', err);
                 // Remove this image if it failed to import/validate.
+                if (!imageSdcDocker) {
+                    cb(err);
+                    return;
+                }
                 var image_uuid = imageSdcDocker.image_uuid;
+                log.error('Unable to import image %s into imgapi: %s',
```

Comment:
Isn't that message a duplicate of [the one above](https://github.com/joyent/sdc-cn-agent/pull/12/files#diff-5a31125d81398aadaf73b9ba01aeb9a5R1046)?